### DISCUSSION TITLE
Distributed Denial Of Service (DDoS) - LinuxGSM server - Left4dead (one).

### DISCUSSION THREAD

**Discussion by guinnessfarts** [GameServerManagers/LinuxGSM#3301] [Q&A] [answered]
Hi. I have a new left 4 dead server setup for the past few weeks(LinuxGSM setup). I see DDoS attacks on many servers in the Left4Dead community. Ruining our games.  Apart from setting a firewall using only ports 27015 on  my dedicated server is there anyway to limit these attacks?

Thank you.

G.

**Comment by dgibbs64** [comment, marked as answer]
Good question. There are options to rate limit ports on your server which you can do. https://docs.linuxgsm.com/linux/firewalls

However, the ideal solution is to have ddos protection sitting in front of your server but that really depends on what your server provider has. So check with them as well

This is the sort of question however that could so with some expanding on in the docs

**Reply by guinnessfarts** [reply]
Ah super. Thanks so much for that. I'll give it a go. Any other ideas?

G.

![Capture](https://user-images.githubusercontent.com/79587294/109016714-56f1b400-76ae-11eb-901a-7371e78f318f.JPG)
